drivers/timers/arch_alarm.c

```diff
@@ -45,6 +45,10 @@
 
 static FAR struct oneshot_lowerhalf_s *g_oneshot_lower;
 
+#ifndef CONFIG_SCHED_TICKLESS
```

Comment:
the visibility is now reduced with ifdef

Reply:
it's same as before:
https://github.com/apache/nuttx/pull/10482/files#diff-0b93d3b636b9a621c7a7904978e06958366a6f72a48a56e954253dbb8e01aa1dR132-R147

Reply:
I see. Any reasons why CI fails?

Reply:
ci fail to download some external package.